common.mk

```diff
@@ -191,6 +190,12 @@ SFC_MFC_TARGETS = \
 
 SFC_REPL_SEQ_MEM = --infer-rw --repl-seq-mem -c:$(MODEL):-o:$(SFC_SMEMS_CONF)
 MFC_BASE_LOWERING_OPTIONS ?= emittedLineLength=2048,noAlwaysComb,disallowLocalVariables,verifLabels,locationInfoStyle=wrapInAtSquareBracket
+ifeq ($(ENABLE_YOSYS_FLOW),)
+	ifneq ($(MFC_BASE_LOWERING_OPTIONS),)
+		MFC_BASE_LOWERING_OPTIONS += ,
+	endif
+	MFC_BASE_LOWERING_OPTIONS += disallowPackedArrays
+endif
```

Comment:
It looks like this flag was already added before: https://github.com/ucb-bar/chipyard/blob/bfa822aec3634d4e455ed9a21f01f18579e08f02/common.mk#L218.

Reply:
Additionally, https://github.com/ucb-bar/chipyard/blob/bfa822aec3634d4e455ed9a21f01f18579e08f02/common.mk#L218 needs to change so that the flag isn't added by default.

Reply:
```
diff --git a/common.mk b/common.mk
index f6bc19c1..50a6b531 100644
--- a/common.mk
+++ b/common.mk
@@ -28,7 +28,7 @@ EXTRA_SIM_CXXFLAGS   ?=
 EXTRA_SIM_LDFLAGS    ?=
 EXTRA_SIM_SOURCES    ?=
 EXTRA_SIM_REQS       ?=
-ENABLE_CUSTOM_FIRRTL_PASS += $(ENABLE_YOSYS_FLOW)
+# ENABLE_CUSTOM_FIRRTL_PASS += $(ENABLE_YOSYS_FLOW)
 
 ifneq ($(ASPECTS), )
        comma = ,
@@ -191,6 +191,12 @@ SFC_MFC_TARGETS = \
 
 SFC_REPL_SEQ_MEM = --infer-rw --repl-seq-mem -c:$(MODEL):-o:$(SFC_SMEMS_CONF)
 MFC_BASE_LOWERING_OPTIONS ?= emittedLineLength=2048,noAlwaysComb,disallowLocalVariables,verifLabels,locationInfoStyle=wrapInAtSquareBracket
+ifneq ($(ENABLE_YOSYS_FLOW),)
+       ifneq ($(MFC_BASE_LOWERING_OPTIONS),)
+               MFC_BASE_LOWERING_OPTIONS := $(MFC_BASE_LOWERING_OPTIONS),
+       endif
+       MFC_BASE_LOWERING_OPTIONS := $(MFC_BASE_LOWERING_OPTIONS)disallowPackedArrays
+endif
 
 # DOC include start: FirrtlCompiler
 # There are two possible cases for this step. In the first case, SFC
@@ -215,11 +221,12 @@ endif
 
 $(MFC_LOWERING_OPTIONS):
        mkdir -p $(dir $@)
-ifeq (,$(ENABLE_YOSYS_FLOW))
        echo "$(MFC_BASE_LOWERING_OPTIONS)" > $@
-else
-       echo "$(MFC_BASE_LOWERING_OPTIONS),disallowPackedArrays" > $@
-endif
+# ifeq (,$(ENABLE_YOSYS_FLOW))
+#      echo "$(MFC_BASE_LOWERING_OPTIONS)" > $@
+# else
+#      echo "$(MFC_BASE_LOWERING_OPTIONS),disallowPackedArrays" > $@
+# endif
 
 $(FINAL_ANNO_FILE): $(EXTRA_ANNO_FILE) $(SFC_EXTRA_ANNO_FILE) $(SFC_LEVEL)
        if [ $(shell cat $(SFC_LEVEL)) = low ]; then jq -s '[.[][]]' $(EXTRA_ANNO_FILE) $(SFC_EXTRA_ANNO_FILE) > $@; fi
```

Reply:
In the `vlsi` directory, I ran `make buildfile tutorial=sky130-openroad` which compiles a TinyRocketConfig, which gave the error: `unsupported packed array expression`. It seems that the `disallowPackedArrays` doesn't work in all cases.

Reply:
@nayiri-k Try with the latest firtool, should be fixed in https://github.com/llvm/circt/pull/6402

Reply:
@abejgonzalez Good news! 1.60.0 is out :-) https://github.com/llvm/circt/releases/tag/firtool-1.60.0